mpyg321/mpyg321.py

```diff
@@ -252,6 +280,22 @@ def jump(self, pos):
         """Jump to position"""
         self.player.sendline("JUMP " + str(pos))
 
+    def volume(self, percent):
+        """Adjust player's volume"""
+        if self.player_version == "mpg123":
+            self.player.sendline("VOLUME {}".format(percent))
+
+    def silence(self):
```

Comment:
After testing, I realize that this command does not put the volume to 0, it only stops the command line output of mpg123. This means that the mpg123 process will stop sending the frame number to the output (which is not displayed to the end user).
Do we really need this? Also, I think the name is a bit confusing for the end user.

Reply:
I agree for the confusion, it should be a private method and force the SILENCE command to mpg123 process by default.

This come from the TIMEOUT problem, my idea behind that was to remove those useless information frames and to don't have to manage them in pexpect to avoid the **TIMEOUT**

Also, pexpect will handle and evaluate every line for nothing, that's overkill I think

Reply:
OK, I think we should keep that optional and activate it by default. 
Let's imagine in the future, that someone wants to build a Ui that shows the advancement of the song, they would need the frame number (that we would need to add to the code).
So basically, we should add to the constructor a "performance mode" argument that would send the SILENCE command.

Reply:
That make sense!